python/pyproject.toml

```
@@ -4,8 +4,28 @@ build-backend = "hatchling.build"
 
 [project]
 name = "remotebmi"
-version = "0.1.0"
-requires-python = ">=3.10"
+description = 'Basic Model Interface over openapi specification'
+readme = "README.md"
+license = "Apache-2.0"
+keywords = [
+    "BMI",
+    "Basic Model Interface",
+    "ewatercycle",
+]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Science/Research",
+    "Programming Language :: Python",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Topic :: Utilities",
+    "Topic :: Scientific/Engineering",
+    "License :: OSI Approved :: Apache Software License"
+]
+requires-python = ">=3.12"
```

Is this correct? Only 3.12 and greater?
```suggestion
requires-python = ">=3.10"
```

---

Otherwise you have to change the classifiers.